arch/common/CMakeLists.txt

```diff
@@ -67,6 +67,10 @@ if (CONFIG_GEN_ISR_TABLES)
   add_dependencies(isr_tables zephyr_generated_headers)
   target_link_libraries(isr_tables zephyr_interface)
   zephyr_library_link_libraries(isr_tables)
+
+  # Do not allow LTO when compiling the isr_tables.c file
+  set_source_files_properties(isr_tabales.c PROPERTIES COMPILE_OPTIONS -fno-lto)
```

Comment:
`tabales` has got to be a typo?

Reply:
Thanks.  That is what I get for a hand merge from the LTS we are using.